tests/features/analytics.feature

```diff
@@ -0,0 +1,19 @@
+@api
+Feature: Webtools Analytics
+  In order to provide analytics
+  As the site manager
+  I need to be able to configure the settings
+
+  @BackupAnalyticsConfigs
+  Scenario: Create Webtools Analytics settings
+    Given I am logged in as a user with the "administer webtools analytics" permission
+    And I am on "admin/config/system/oe_webtools_analytics"
```

Comment:
We should use the transformation context to turn this path into a business readable string, such as:

```
    And I am on "the Webtools Analytics configuration page"
```

And then in the `behat.yml.dist`:
```
      - OpenEuropa\Behat\TransformationContext:
          pages:
            Webtools Analytics configuration: "/admin/config/system/oe_webtools_analytics"
```

Reply:
This does not work for me.